twitter-langchain/examples/chatbot/README.md

```diff
@@ -0,0 +1,51 @@
+# CDP Agentkit Twitter Langchain Extension Examples - Chatbot
+
+This example demonstrates an agent setup as a terminal style chatbot with access to Twitter (X) API actions.
+
+## Ask the chatbot to engage in the Twitter (X) ecosystem!
+- "Transfer a portion of your ETH to john2879.base.eth"
```

Comment:
this doesn't work without access to other actions right?

Reply:
hah right, i left it in as tribute i suppose